### ISSUE TITLE
Differences between released model and training script

### ISSUE THREAD

**Issue by MoustafaMeshry** [junyanz/BicycleGAN#50]
First, thanks for releasing a clean and well trained code for your paper!
I tried retraining the edges2shoes model using the "scripts/train_edges2shoes.sh" script. However, the validation results generated by my retrained model look clearly inferior to results from your released trained model. So, are there any extra tricks used for the released trained model (e.g. longer training, different hyper parameter or loss settings, ... etc)?
Details: I trained using a single Quadro P6000 GPU (with cuda 9.0 and python 2.7) for the full 60 epochs without changing your code/scripts.

**Comment by junyanz** [comment]
We didn't use a perceptual loss, longer training, or hyperparameter tuning. I did update the code a couple of times to make it compatible with PyTorch new version and consistent with pix2pix/cyclegan. I will recommend that you try NITER=25 and NITER_DECAY=25. I found training logs with both this parameter configuration and also (NITER=30 and NITER_DECAY=30).
